devices/src/virtio/net.rs

```diff
@@ -726,6 +741,16 @@ impl Net {
             allow_mmds_requests,
         )
     }
+
+    fn get_guest_mac(&self) -> Option<MacAddr> {
```

Comment:
```suggestion
    fn guest_mac(&self) -> Option<MacAddr> {
```

Reply:
Right - more in line with the rest of the project. Done.